packages/riverpod_lint/bin/custom_lint.dart

```diff
@@ -15,8 +15,14 @@ import 'package:riverpod_lint/src/type_checker.dart';
 
 const _providerBase =
     TypeChecker.fromName('ProviderBase', packageName: 'riverpod');
-const _autoDispose =
-    TypeChecker.fromName('AutoDisposeProviderBase', packageName: 'riverpod');
+const _nonAutoDispose = TypeChecker.any([
+  TypeChecker.fromName('AlwaysAliveProviderBase', packageName: 'riverpod'),
```

Comment:
This line is redundant, considering `AlwaysAliveProviderBase` is an `AlwaysAliveProviderListenable`